techniques/system/server-roles/1.0/password-check.cf

```diff
@@ -295,9 +236,13 @@ bundle agent root_password_check_dav
     !SuSE::
       "htpasswd_bin" string => "/usr/bin/htpasswd";
 
+    root_server::
+    # Build an array using webdav configuration values
+      "rudder[rudder.webdav.password]" string => "${p.dav_password[2]}";
+
   classes:
 
-      "dav_cant_connect" not => returnszero("${g.rudder_curl} --tlsv1.2 --proxy '' ${g.rudder_verify_certs_option} --silent --fail --output /dev/null --user ${g.davuser}:${g.davpw} --upload-file /opt/rudder/etc/uuid.hive https://localhost/inventory-updates/uuid.hive","noshell");
+      "dav_cant_connect" not => returnszero("${g.rudder_curl} --tlsv1.2 --proxy '' ${g.rudder_verify_certs_option} --silent --fail --output /dev/null --user ${g.davuser}:${p.dav_password[2]} --upload-file /opt/rudder/etc/uuid.hive https://localhost/inventory-updates/uuid.hive","noshell")
```

Comment:
Could not find where p.dav_password was defined

Reply:
defined in a getfields